pkg/ebpf/events/derived/derived.go

```diff
@@ -0,0 +1,29 @@
+package derived
```

Comment:
So I like having it as a subpackage, but there's barely anything here. 
The only exported symbol that's used from here is the DeriveFn, I don't see how that's worthy of it's own package. If more stuff was moved here, or the other exported symbols would be actually used, then it would be much more justified (which it could be).